network-protection/network-protection-impl/src/main/res/values/donottranslate.xml

```diff
@@ -126,6 +126,10 @@
     <!-- VPN settings -->
     <string name="netpVpnSettingKillSwitch">Always On</string>
     <string name="netpVpnSettingAlwaysOn">Always On</string>
+    <string name="netpManageRecentAppsProtectionAllowUnrestrictedBattTitle">Allow Unrestricted App Battery Usage</string>
+    <string name="netpManageRecentAppsProtectionAllowUnrestrictedBattByline">In App Battery Settings, allow Unrestricted usage to prevent Android from disabling App Tracking Protection in the background</string>
```

Comment:
I think this should be Network Protection than App Tracking Protection

Reply:
done